scribe/__init__.py

```diff
@@ -12,6 +14,16 @@
 app = Flask(__name__)
 app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///../data/temp.db'
 
+class JSONSecureCookieSession(SecureCookieSession):
+    serialization_method = json
+
+class JSONSecureCookieSessionInterface(SecureCookieSessionInterface):
+    session_class = JSONSecureCookieSession
+
+app.secret_key = 'foobar'
```

Comment:
Yeah it's bad practice to have it inline, but tbh I don't think it really matters for us. But point taken.